charts/m4d/values.yaml

```diff
@@ -12,19 +24,110 @@ global:
   # Default behavior: latest images will be Always else IfNotPresent.
   # ref: https://kubernetes.io/docs/concepts/containers/images/#updating-images
   imagePullPolicy: ""
-  # Reference to one or more secrets to be used when pulling images
+  # Reference to one or more secrets to be used when pulling images.
   # ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   imagePullSecrets: []
+  # Default connection timeout for GRPC connections.
+  connectionTimeout: 120
+
+# Cluster metadata values
+cluster:
+  # Set to the name of the cluster.
+  name: hobbiton
+  # Set to cluster region.
+  region: eriador
+  # Set to cluster zone.
+  zone: theshire
+  # Set to the cluster Vault auth method path.
+  vaultAuthPath: kubernetes
+
+# Configuration when deploying to a coordinator cluster.
+coordinator:
+  # Set to false to disable coordinator components in manager.
+  enabled: true
+
+  # Configures the catalog system name to be used by the coordinator manager.
+  # Accepted values are "katalog", "egeria" or any meaningful name if a third party connector is used.
+  catalog: "katalog"
+
+  # Overrides the catalog connector URL.
+  # Defaults to `<catalog>-connector`.
+  catalogConnectorURL: ""
```

Comment:
How does helm 3 deal with defaults? Is an empty string treated the same as nil or no definition? Thus when defining `""` here is the default executed or is the empty string written in as value?

Reply:
generally they are treated the same. When you use `default` in the templates any of these values are replaced with the default.